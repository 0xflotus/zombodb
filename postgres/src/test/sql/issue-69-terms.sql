SELECT *
FROM zdb_tally('so_posts', 'title', '^A.*', '', 5000, 'term')
LIMIT 10;