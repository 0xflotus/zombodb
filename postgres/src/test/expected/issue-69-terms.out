SELECT *
FROM zdb_tally('so_posts', 'title', '^A.*', '', 5000, 'term')
LIMIT 10;
   term   | count  
----------+--------
          | 129680
 A        |   9898
 A.APPEND |      1
 A.I      |      3
 A7       |      1
 AA       |      1
 AAC      |      1
 AARDVARK |      1
 AB       |      1
 ABALONE  |      1
(10 rows)

